client/src/views/Footer.js

```diff
@@ -0,0 +1,17 @@
+import React, { Component } from 'react';
+import '../css/Footer.css';
+
+export default class Footer extends Component {
+  render() {
+    return (
+      <div className="Footer">
+        <a href="https://www.ziprecruiter.com/jobs" id="jobs_widget_link">
+        <span>Job Search by</span>
+          <span id="zr_logo_container">
+            <img id="zr_logo" src="https://www.ziprecruiter.com/img/logos/logo-sm-black-304px.png" alt="ZipRecruiter" width="120" />
```

Comment:
It would be better if we download the image under assets/ folder and reference it from there; in case they change the URL :)